packages/renderer/src/lib/ui/NavPage.svelte

```diff
@@ -5,19 +5,19 @@ export let searchEnabled = true;
 </script>
 
 <div class="flex flex-col w-full h-full shadow-pageheader">
-  <div class="flex flex-col w-full h-full pt-4">
-    <div class="flex pb-2">
+  <div class="flex flex-col w-full h-full pt-4" role="region" aria-label="{title}">
+    <div class="flex pb-2" role="region" aria-label="header">
```

Comment:
hi, not sure why we do need a role for the header itself ?

Reply:
I would like to divide the page into three "frontmost" areas/regions that would help us to locate inside. hope that make sense this way.
![ui-aria-pages](https://github.com/containers/podman-desktop/assets/19164299/70aedf93-2297-4eb0-99a5-9065ef3d5b13)

Reply:
> hi, not sure why we do need a role for the header itself ?

And to make the answer more precise, in the same region (header one) there are also additional actions buttons. 
`this.additionalActions = this.header.getByRole('group', { name: 'additionalActions' });`